### PR TITLE
fix example - don't stringify err

### PR DESCRIPTION
**Pull request by derrickstaten**
@dmitrig01 this does not include the request header change needed to trigger the `js_response` code path on link-service

### CODE REVIEW

**Comment by dmitrig01** [comment]
you need to edit example.template.html and then make

**Comment by derrickstaten** [comment]
@dmitrig01 updated

**Comment by dmitrig01** [comment]
:+1:

File: src/6_branch.js [hunk 1]
```diff
@@ -922,7 +922,7 @@ Branch.prototype['link'] = wrap(callback_params.CALLBACK_ERR_DATA, function(done
  * ##### Example
  * ```js
  * branch.sendSMS(
- *     phone: '9999999999',
+ *     '9999999999',
```
Comment (Kirkkt) [inline src/6_branch.js]: good! please run `make` after changing this, which will update docs/web/3_branch_web.md

**Comment by Kirkkt** [comment]
:+1: except the above knit

**Pull request merged**